pkg/backend/organizations_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2016-2025, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend
+
+import (
+	"context"
+	"testing"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDefaultOrg(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	userConfiguredOrg := "user-configured-default-org"
+	backendConfiguredOrg := "backend-configured-org"
+	t.Run("prefers user-configured default org", func(t *testing.T) {
+		t.Parallel()
+
+		// GIVEN
```

Comment:
We call these `Arrange`/`Act`/`Assert` in our code normally, so if we want to keep them we should use those for consistency.  (Personally I'd rather just see these comments not there, since I don't think they really add much value and make the tests harder to read)